app/scripts/views/mixins/password-prompt-mixin.js

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Plugin to display hints for passwords,
+ * hints change as the user types
+ */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  var BaseView = require('views/base');
+  var t = BaseView.t;
+
+  var Tooltip = require('views/tooltip');
+
+  var TOOLTIP_MESSAGES = {
+    FOCUS_PROMPT_MESSAGE: t('8 characters minimum, but longer if you plan to sync passwords.'),
+    INITIAL_PROMPT_MESSAGE: t('A strong, unique password will keep your Firefox data safe from intruders.'),
+    WARNING_PROMPT_MESSAGE: t('This is a common password; please consider another one.')
+  };
+
+  var PasswordPromptMixin = {
+    // only the .check-password fields will be checked
+    events: {
+      'blur .check-password': 'onPasswordBlur',
+      'focus .check-password': 'onInputFocus',
+      'keyup .check-password': 'onInputKeyUp'
+    },
+
+    afterRender: function () {
+      this.updateFormValueChanges();
+    },
+
+    displayInitialPrompt: function (inputEl) {
+      this.$el.find(inputEl).siblings('.input-help-focused').html(TOOLTIP_MESSAGES.INITIAL_PROMPT_MESSAGE);
```

Comment:
could you make `'.input-help-focused'` "constants" above just like `TOOLTIP_MESSAGES` above (except do not make it an object.